src/android/wxapi/EntryActivity.java

```diff
@@ -1,4 +1,4 @@
-package __PACKAGE_NAME__.wxapi;
+package xu.li.cordova.wechat.wxapi;
```

Comment:
This was getting replaced at build time with `com.smartly.hybrid`. My first attempt to fix this was to simply include this code as part of that package and under the same namespace. 

However, this code has a dependence on Cordova, which is why it ends up in `capacitor-cordova-android-plugins`. The `com.smartly.hybrid` namespace doesn't actually have access to Cordova.